placenames.mss

```diff
@@ -369,6 +369,38 @@
   }
 }
 
+#placenames-small::quarter {
+  [place = 'quarter'] {
+    [zoom >= 14][zoom < 17] {
+      text-name: "[name]";
+      text-fill: @placenames;
+      text-face-name: @book-fonts;
+      text-halo-fill: @standard-halo-fill;
+      text-halo-radius: @standard-halo-radius * 1.5;
+      [zoom >= 14] {
+        text-halo-fill: white;
+        text-size: 11;
+        text-wrap-width: 65; // 5.0 em
+        text-line-spacing: -0.65; // -0.05 em
+        text-margin: 9.1; // 0.7 em
+      }
+      [zoom >= 15] {
+        text-fill: @placenames-light;
+        text-size: 12;
+        text-wrap-width: 70; // 5.0 em
+        text-line-spacing: -0.70; // -0.05 em
+        text-margin: 9.8; // 0.7 em
+      }
+      [zoom >= 16] {
+        text-size: 14;
```

Comment:
With the `em` value as starting point, the following `text-wrap-width`, `text-line-spacing` and `text-margin` absolute values would have to be recalculated for _14 em⁻¹_.